### PR TITLE
OOP school library: Add basic UI

### PR DESCRIPTION
**Pull request by gitbygaurav**
###Hii CR, 👋

LifeCollegeGIF

In this PR, I have implemented the following changes:-

- [x] Created Basic UI of school library application.
- [x] When you run the main.rb file on the console, you will have choices to:
   1. list all books.
   2. list all people.
   3. create a student or a teacher
   4. create a book.
   5. create a rental.
   6. list all rentals for a given person's id.       

Please review my changes,
Thanks🍻

### CODE REVIEW

File: rental.rb [hunk 1]
```diff
@@ -1,14 +1,11 @@
...
   def initialize(date, person, book)
     @date = date
-
     @person = person
-    person.rentals << self
-
     @book = book
+    person.rentals << self
     book.rentals << self
   end
```
Comment (ghost) [inline rental.rb:10]: - [ ] When I try to create a rental, I get an error. Kindly check it out

image
image

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @gauravgangwarweb , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights

✅ Correct use of Gitflow
✅ No linter error
✅ UI is being displayed 👍 



## Required Changes ♻️

- [ ] Kindly fix the link to your `MIT` License as it raises a `404` page error when visisted.
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by m05e5 — APPROVED** [review]
**Hi @gauravgangwarweb 
You did great. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
CleverScornfulBuffalo-size_restricted

## Highlights

✔️ Clean and well-organized code
✔️ No Linters error
✔️ Good PR description
## Optional suggestions


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**m05e5**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._**

**Comment by gitbygaurav** [comment]
Hii @m05e5 
Thanks for approving my PR.

**Pull request merged**